app/src/main/res/values/strings.xml

```diff
@@ -112,6 +136,30 @@
     <string name="show_mod_bold">Признак жирного шрифта</string>
     <string name="show_mod_italic">Признак курсивного шрифта</string>
 
+    <string name="show_music_note_c">Нота ДО</string>
+    <string name="show_music_note_d">Нота РЕ</string>
+    <string name="show_music_note_e">Нота МИ</string>
+    <string name="show_music_note_f">Нота ФА</string>
+    <string name="show_music_note_g">Нота СОЛЬ</string>
+    <string name="show_music_note_a">Нота ЛЯ</string>
+    <string name="show_music_note_b">Нота СИ</string>
+
+    <string name="show_music_rest_8th">Пауза длительности одна восьмая</string>
+    <string name="show_music_rest_4th">Пауза длительности одна четверть</string>
+    <string name="show_music_rest_half">Пауза половинной длительности</string>
+    <string name="show_music_rest_full">Целая пауза</string>
+    <string name="show_music_octave1">Обозначение первой октавы</string>
+    <string name="show_music_octave2">Обозначение второй октавы</string>
+    <string name="show_music_octave3">Обозначение третьей октавы</string>
+    <string name="show_music_octave4">Обозначение четвёртой октавы</string>
+    <string name="show_music_octave5">Обозначение пятой октавы</string>
+    <string name="show_music_octave6">Обозначение шестой октавы</string>
+    <string name="show_music_octave7">Обозначение седьмой октавы</string>
+    <string name="show_music_hyphen">Музыкальный знак переноса</string>
+    <string name="show_music_sharp">Диез ♯</string>
```

Comment:
Are these symbols normally read by the talkback?

Reply:
Thank you for this comment. I thought previously that they are ignored by the TalkBack because Samsung TTS ignores them, but after your question, I tried Google TTS and found that it reads sharp and flat signs twice. I'd better remove these symbols

Reply:
Can they only be removed for the talkback, but stay visible?

Reply:
Yes, but it will require a few time to code. I shall add it to "things we will probably never do"

Reply:
removed symbols & added a point to #257